composer.lock

```diff
@@ -3427,7 +3427,7 @@
                 "MIT"
             ],
             "description": "eLife Sciences API validator",
-            "time": "2017-09-06T06:41:40+00:00"
+            "time": "2018-03-15T10:16:56+00:00"
         },
```

Comment:
cross-check: no update of `elife/api` (which is `api-raml`) or `elife/patterns` required?

Reply:
`elife/patterns` shouldn't be updated because the images were not used in https://elifesciences.org/annual-reports (nor anywhere else I know about)

Reply:
> cross-check: no update of elife/api (which is api-raml) or elife/patterns required?

`elife/api` appears to already be up to date. `elife/patterns` doesn't require any change.

Reply:
and `elife/api` is already at the newest one, 3636c5ceea8bdf5fa50902a9b6940cb42bc44138 so doesn't need updating